src/app/counter/counter-booking/counter-booking.component.ts

```diff
@@ -12,6 +12,9 @@ import { Elements, Element, StripeService } from "ngx-stripe";
   styleUrls: ["./counter-booking.component.css"],
 })
 export class CounterBookingComponent implements OnInit {
+  minPrice = 0;
+  maxPrice = 10000;
+  customPrice = 10000;
```

Comment:
this number should be taken either from a constants file, or the greatest price in the list of flights